example_configs/jellyfin.md

```diff
@@ -37,9 +37,9 @@ Otherwise, just use:
 ```
 ### Admin Base DN
 
-The DN of your admin group. If you have `media_admin` as your group you would use:
+The DN to search for your admins. If your admins are also users use:
```

Comment:
Erm, the admins are all users, no?

Reply:
Well technically yes, but not strictly. You could have a user who is a member of the admin group but not a member of the user group.

You could also have no user group set at all and only designate an admin group.

Reply:
I mean, admins are LDAP users, and all users are in that base DN, so you'll always need this value.

Also, regarding what @xavierus666 said, I think even admins have to pass a user check when logging in.

Reply:
Fair. I have updated the language here.